definitions/divorce/json/CaseField/CaseField-shareACase.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "LiveFrom": "18/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ApplicantOrganisationPolicy",
```

Comment:
ID (and name) should be: PetitionerOrganisationPolicy and Petitioner Organisation Policy